most_dominant.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""Given a winner, what was the closest game they played in the tournament?"""
+
+import sys
+
+from utils import sum_for_game, get_flattened_games
+
+def get_winner(year, game_years):
```

Comment:
Your `game_years` construct is working against you here. The year and winner are readily available if you go directly from the JSON files. (The year is always `data['year']` and the championship is always `data['finalfour'][2]`).